docs/script.rst

```diff
@@ -714,80 +705,78 @@ Table of optional parameters:
+overwrite           When ``true`` overwrite any exiting file with the same name, otherwise throw a `FileAlreadyExistsException <http://docs.oracle.com/javase/8/docs/api/java/nio/file/FileAlreadyExistsException.html>`_ (default: ``false``)
 ==================  ================
 
 
 Copy files
 ----------
 
-The method ``copyTo`` allows you to copy a file into a new file or into a directory, or copy a directory to a new
-directory. Having a file variable ``myFile``, the following example shows how to copy a file into a new file
-with a different name::
+The method ``copyTo`` copies a file into a new file or into a directory, or copie a directory to a new
+directory::
 
   myFile.copyTo('new_name.txt')
 
 
-.. note:: If the target file already exists, it will be replaced by the new one. Note also that if the target is
-  a directory, the source file will be copied into that folder maintaining the original name.
-
+.. note:: If the target file already exists, it will be replaced by the new one. Note also that, if the target is
+  a directory, the source file will be copied into that directory, maintaining the file's original name.
 
-When the source file is a directory, all the directory content is copied to the target folder. For example::
 
+When the source file is a directory, all its content is copied to the target directory::
 
   myDir = file('/some/path')
   myDir.copyTo('/some/new/path')
 
 
-If the target path does not exist, it will be created automatically.
+  If the target path does not exist, it will be created automatically.
 
-.. tip:: The ``copyTo`` method mimics the semantic of the Linux command ``cp -r <source> <target>``,
-  with the following caveats: Unix BASH distinguish between paths having or not having and ending slash, for example:
-  ``/some/path/name`` and ``/some/path/name/``. The first locates a regular file while the latter identifies a directory
-  location. With Nextflow, due to Java files API implementation, this is not possibles and both strings represents the same path.
-  If that path exists on the file systems it is handled accordingly (as a regular file or as a directory). If the path does not
-  exist, it is supposed to locate a regular file (and any parent directory will be created automatically).
+.. tip:: The ``copyTo`` method mimics the semantics of the Linux command ``cp -r <source> <target>``, with the
+         following caveat: While Linux tools often treat paths ending with a slash (e.g. ``/some/path/name/``)
+         as directories, and those not (e.g. ``/some/path/name``) as regular files, Nextflow (due to its use of
+         the Java files API) views both these paths as the same file system object. If the path exists, it is
+         handled according to its actual type (i.e. as a regular file or as a directory). If the path does not
+         exist, it is treated as a regular file, with any missing parent directories created automatically.
```

Comment:
I found this paragraph confusing and did my best to improve it and make it accurate.

Reply:
👍